styleguide/source/_patterns/04-templates/three-column.twig

```diff
@@ -0,0 +1,38 @@
+{% block skipNav %}
+  {% include "@atoms/skip-nav.twig" %}
+{% endblock %}
+
+<div class="container">
+  {% block ribbon %}
```

Comment:
Are we sure it makes sense to show these blocks here in the Template-level pattern? Should we only show the "three column" parts here?

Reply:
I put the ribbon and header blocks here so we can override them later. And to show what the actual template will look like as oppose to in pieces